File: Step 3: Solve Problems on Arrays/Step 3.1: Easy/LargestElementInArray.cpp
#include <iostream>
using namespace std;

int LargestElementInArray(int arr[], int n) {
    int max = arr[0];
    for (int i = 0; i < n; i++)
        if (arr[i] > max) max = arr[i];

    return max;
}

int main() {
    int n;
    cin >> n;
    int arr[n];
    for (int i = 0; i < n; i++) cin >> arr[i];
    cout << LargestElementInArray(arr, n);
    return 0;
}

File: Step 3: Solve Problems on Arrays/Step 3.1: Easy/LargestElementInArray.py
def LargestElementInArray(arr, n):
    max = arr[0]
    for i in range(0, n):
        if arr[i] > max:
            max = arr[i]
    return max


if __name__ == "__main__":
    n = int(input())
    arr = [int(x) for x in input().split()]
    print(LargestElementInArray(arr, n))
    print("")
